WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -188,11 +187,14 @@ private void moveToStats() {
     }
 
     private void navigateStats() {
-        swipeDownOnView(R.id.scroll_view);
         moveToStats();
 
         swipeToAvoidGrayOverlay(R.id.statsPager);
 
+        if (isElementDisplayed(R.id.button_negative)) {
```

Comment:
It looks like `clickOn` is already calling `waitForElementToBeDisplayed`, so these `isElementDisplayed` checks seems unnecessary.

Reply:
Ahhh so the thing here is that this button may or may not be present, it's not consistently shown, and I'd only like to click on it if it happens to be there. The `waiton` will fail if it isn't present.